packages/amplify-graphql-api-construct/src/amplify-graphql-api.ts

```diff
@@ -113,4 +122,24 @@ export class AmplifyGraphqlApi extends Construct {
 
     this.resources = generateConstructExports(transformedResources.rootStack, transformedResources.stacks, transformerStack);
   }
+
+  /**
+   * Get the function slots generated by the GraphQL transform operation, adhering to the FunctionSlot interface.
+   * @returns the list of generated function slots in the transformer, in order to facilitate overrides.
+   */
+  getGeneratedFunctionSlots(): FunctionSlot[] {
+    return Object.entries(this.transformerGeneratedResolvers)
+      .filter(([name]) => name.split('.').length === 6)
```

Comment:
Some field level resolvers like `Model.Field.vtl` do not confirm to this format. Are we leaving them out here?